pkg/hostagent/dns/dns.go

```diff
@@ -99,14 +99,14 @@ func NewHandler(opts HandlerOptions) (dns.Handler, error) {
 		clientConfig: cc,
 		clients:      clients,
 		IPv6:         opts.IPv6,
-		cname:        make(map[string]string),
-		ip:           make(map[string]net.IP),
+		cnameToHost:  make(map[string]string),
+		hostToIP:     make(map[string]net.IP),
 	}
 	for host, address := range opts.StaticHosts {
 		if ip := net.ParseIP(address); ip != nil {
-			h.ip[host] = ip
+			h.hostToIP[host] = ip
 		} else {
-			h.cname[host] = limayaml.Cname(address)
+			h.cnameToHost[host] = limayaml.Cname(address)
```

Comment:
I think I remember that this one reference to `limayaml` pulls in a whole bunch of other packages into the downstream project's `go.mod`. Maybe it doesn't matter, as the unused code won't be included in the binary, but I felt that maybe moving `Cname` into the `dns` package would be cleaner.

This can be done in a followup PR, if this should be done at all. Opinions?

Reply:
Sure, that makes sense to put it under the `dns` package.

Reply:
I understand this has not been done because just moving the function over would create an import loop. Not really important anyways; we can always changes this later.